scripts/copy-code-to-clipboard.js

```diff
@@ -0,0 +1,26 @@
+const Clipboard = require('clipboard')
+
+new Clipboard('.btn-clipboard') // eslint-disable-line
+
+module.exports = function copyCodeToClipBoard () {
+  document.querySelectorAll(`code.hljs`).forEach(code => {
+    let id = `_${Math.random().toString(36).substr(3)}`
+    code.id = id
+    let copyButton = document.createElement('button')
+    copyButton.classList.add('btn-clipboard', 'simptip-position-top')
+    copyButton.setAttribute('data-tooltip', 'Copy to clipboard')
+    copyButton.setAttribute('data-clipboard-target', `#${id}`)
+    copyButton.innerHTML = 'Copy'
+    code.parentElement.appendChild(copyButton)
+  })
+  document.querySelectorAll('.btn-clipboard').forEach(button => {
```

Comment:
Using Cliboard's event handlers might make this a little cleaner: https://github.com/zenorocha/clipboard.js#events

Also it would mean we'd have a `clipboard` instance and could get rid of the `// eslint-disable-line` comment.

Reply:
This `disable` for `Do not use 'new' for side effects.` :(

Reply:
I know! I just meant you could do this:

```js
const clipboard = new Clipboard('.btn-clipboard')

clipboard.on('success', (event) {
    console.info('Action:', e.action)
    console.info('Text:', e.text)
    console.info('Trigger:', e.trigger)
    e.clearSelection()
})
```